src/finch/compiled.py

```diff
@@ -27,8 +27,23 @@ def lazy(tensor: Tensor):
         return Tensor(jl.Finch.LazyTensor(tensor._obj))
     return tensor
 
-
-def compute(tensor: Tensor, *, verbose: bool = False):
+def set_optimizer(opt="default"):
+    if opt == "default":
+        jl.Finch.set_scheduler_b(jl.Finch.default_scheduler())
+    elif opt == "galley":
+        jl.Finch.set_scheduler_b(jl.Finch.galley_scheduler())
+    return
+
+def clear_optimizer_cache():
+    jl.empty_b(jl.Finch.codes)
+    return
+
+def compute(tensor: Tensor, *, verbose: bool = False, opt=""):
```

Comment:
Let's also add `opt` parameter to the `def compiled` decorator above so we can use it:

```python
@compiled(opt="galley")
def sddmm(s, a, b):
    s * (a @ b)

```

Reply:
Done.